home/tests.py

```diff
@@ -96,3 +94,43 @@ def question_tag_test_data(question_test_data):
     new_pair.tag = test_tag
     new_pair.save()
     return test_tag
+
```

Comment:
You'd probably want to group tests in different classes

Reply:
@ifireball
Done